presto-orc/src/test/java/com/facebook/presto/orc/TestSelectiveOrcReader.java

```diff
@@ -352,6 +352,9 @@ public void testArrays()
     {
         Random random = new Random(0);
 
+        tester.testRoundTrip(arrayType(VARCHAR),
+                ImmutableList.of(ImmutableList.of()), IS_NOT_NULL);
```

Comment:
put these two lines into one

Reply:
actually i don't need this test, this was a temporary change for a different issue, will remove it